gradle.properties

```diff
@@ -34,4 +34,5 @@ slf4jVersion=1.7.25
 spectatorVersion=0.61.0
 typesafeConfigVersion=1.3.3
 zipkinSenderVersion=2.7.6
-springBootDependenciesVersion=2.1.5.RELEASE
+springBootDependenciesVersion=2.2.0.BUILD-SNAPSHOT
```

Comment:
I guess this should go to 1.7.0 I don't think we want to deliver that to 1.6.x since it brakes backward compatibility with elder springs (at least it should do that).